events/smart-descriptions.json

```diff
@@ -3655,5 +3655,90 @@
         "value": "changed-audit-configuration"
       }]
     }
+  ],
+  "mcafee web gateway": [
+    {
+      "value": "Connection from {source.ip} to {url.domain} for user {user.name} blocked: {event.reason}",
+      "conditions": [{
+        "field": "event.action",
+        "value": "denied"
+      },{
+        "field": "user.name"
+      }
+      ],
+      "relationships": [{
+        "source": "source.ip",
+        "target": "url.domain",
+        "type": "denied"
+      }]
+    },
+    {
+      "value": "Connection from {source.ip} to {url.domain} blocked: {event.reason}",
+      "conditions": [{
+        "field": "event.action",
+        "value": "denied"
+      }
+      ],
+      "relationships": [{
+        "source": "source.ip",
+        "target": "url.domain",
+        "type": "denied"
+      }]
+    },
+    {
+      "value": "Connection from {source.ip} to {url.domain} for user {user.name} allowed",
+      "conditions": [{
+        "field": "event.action",
+        "value": "allowed"
+      },{
+        "field": "user.name"
+      }
+      ],
+      "relationships": [{
+        "source": "source.ip",
+        "target": "url.domain",
+        "type": "allowed"
+      }]
+    },
+    {
+      "value": "Connection from {source.ip} to {url.domain} allowed",
```

Comment:
```suggestion
      "value": "Connection from {source.ip} to {url.domain} allowed.",
```

Reply:
I don't think the dot at the end of the string is required.